deploy/gcloud/Dockerfile.consumer

```diff
@@ -46,8 +46,8 @@ FROM alpine:latest as certs
 
 RUN apk --update add ca-certificates
 
-# Build a small executable from docker scratch.
-FROM scratch
+# Build a small executable from docker alpine.
```

Comment:
Explain why alpine is necessary here instead of scratch.

Reply:
Done